cypress/integration/tests/developer/applicationinventory/assessment/create.test.ts

```diff
@@ -150,4 +155,26 @@ describe("Application validations", { tags: "@tier2" }, () => {
         cy.wait("@getApplication");
         notExists(application.name);
     });
+
+    it("Bulk deletion of applications", function () {
+        // Click dropdown toggle button to make 'Select page' selection.
+        cy.get("button[aria-label='Select']").click();
+        cy.get("ul[role=menu] > li").contains("Select all").click();
+
+        // Select all the applications on the current page and delete the applications
+        cy.get("ul[role=menu] > li").contains("Select page").click();
+        cy.get(actionButton).eq(1).click();
+        cy.get("a.pf-c-dropdown__menu-item").contains("Delete").click();
+        clickByText(button, "Delete");
+
+        applicationList = createMultipleApplications(2);
+        // Click dropdown toggle button to make 'Select all' selection.
+        cy.get("button[aria-label='Select']").click();
+
+        // Select all applications and delete them
+        cy.get("ul[role=menu] > li").contains("Select all").click();
+        cy.get(actionButton).eq(1).click();
+        cy.get("a.pf-c-dropdown__menu-item").contains("Delete").click();
+        clickByText(button, "Delete");
```

Comment:
Let's assert if the applications are deleted . 
So if you create 11 applications and do select page and delete , you can assert if 1 rows is present in application table .

And when you selectall and delete , you can assert that 0 applications are present.

Reply:
Done